README.md

```diff
@@ -192,6 +192,9 @@ limitations under the License.
 
 ## Release Notes
 
+### v1.2.2
```

Comment:
Please add a release note for v1.2.1 which I forgot to do before publishing it:

### v1.2.1

- converted all unit tests from nodeunit to jest

Reply:
I updated it